sdk/resource/internal/schema/schema.go

```diff
@@ -0,0 +1,1030 @@
+// Code generated by "go.opentelemetry.io/otel/sdk/resource/internal/schema/gen"; DO NOT EDIT.
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package schema
+
+import (
+	ast10 "go.opentelemetry.io/otel/schema/v1.0/ast"
+	types10 "go.opentelemetry.io/otel/schema/v1.0/types"
+	"go.opentelemetry.io/otel/schema/v1.1/ast"
+	"go.opentelemetry.io/otel/schema/v1.1/types"
+)
+
+var Schemas = map[string]*ast.Schema{
+	"https://opentelemetry.io/schemas/1.4.0": {
+		FileFormat: "1.0.0",
+		SchemaURL:  "https://opentelemetry.io/schemas/1.4.0",
+		Versions: map[types.TelemetryVersion]ast.VersionDef{
+			"1.4.0": {},
+		},
+	},
+	"https://opentelemetry.io/schemas/1.5.0": {
+		FileFormat: "1.0.0",
+		SchemaURL:  "https://opentelemetry.io/schemas/1.5.0",
+		Versions: map[types.TelemetryVersion]ast.VersionDef{
+			"1.4.0": {},
+			"1.5.0": {},
+		},
+	},
+	"https://opentelemetry.io/schemas/1.6.1": {
+		FileFormat: "1.0.0",
+		SchemaURL:  "https://opentelemetry.io/schemas/1.6.1",
+		Versions: map[types.TelemetryVersion]ast.VersionDef{
+			"1.4.0": {},
+			"1.5.0": {},
+			"1.6.1": {},
+		},
+	},
+	"https://opentelemetry.io/schemas/1.7.0": {
+		FileFormat: "1.0.0",
+		SchemaURL:  "https://opentelemetry.io/schemas/1.7.0",
+		Versions: map[types.TelemetryVersion]ast.VersionDef{
+			"1.4.0": {},
+			"1.5.0": {},
+			"1.6.1": {},
+			"1.7.0": {},
+		},
+	},
+	"https://opentelemetry.io/schemas/1.8.0": {
+		FileFormat: "1.0.0",
+		SchemaURL:  "https://opentelemetry.io/schemas/1.8.0",
+		Versions: map[types.TelemetryVersion]ast.VersionDef{
+			"1.4.0": {},
+			"1.5.0": {},
+			"1.6.1": {},
+			"1.7.0": {},
+			"1.8.0": {
+				Spans: ast10.Spans{
```

Comment:
Is there a reason we have all the Span and Metrics changes?  This seems to be solely focused on resources, and only capturing the changes for resources looks to reduce most of this structure.